builds/jazz-build-module/sonar-module.groovy

```diff
@@ -40,9 +42,33 @@ def configureSonarProperties() {
 		g_sonar_project_properties["sonar.projectName"] = g_sonar_projectKey
 		g_sonar_project_properties["sonar.projectVersion"] = g_sonar_projectVersion
 		g_sonar_project_properties["sonar.sources"] = g_sonar_sources
+		g_sonar_project_properties["sonar.exclusions"] = g_sonar_exclusions
 
 		if (service_config['runtime'].indexOf("java") > -1) {
 			g_sonar_project_properties["sonar.java.binaries"] = g_sonar_java_binaries
+			g_sonar_project_properties["sonar.coverage.jacoco.xmlReportPaths"] = g_sonar_java_coverage_reports
+		}
+
+		if (service_config['runtime'].indexOf("go") > -1) {
+			g_sonar_project_properties["sonar.tests"] = "."
+			g_sonar_project_properties["sonar.test.inclusions"] = "**/*_test.go"
+			g_sonar_project_properties["sonar.go.coverage.reportPaths"] = "cov.out"
+		}
+
+		if (service_config['runtime'].indexOf("node") > -1) {
```

Comment:
Please put a check for runtime just like how you have added for framework below

Reply:
Runtime is always present in the config file.
Framework is present only in case of Angular/React/HTML website.